test/cpp/api/serialize.cpp

```diff
@@ -100,7 +100,8 @@ void test_serialize_optimizer(DerivedOptimizerOptions options) {
     optimizer.zero_grad();
     auto y = model->forward(x).sum();
     y.backward();
-    optimizer.step();
+    auto closure = []() { return torch::tensor({10}); };
+    optimizer.step(closure);
```

Comment:
Wondering would it work if we do `optimizer.step(nullptr)` instead? It seems to be simpler than having to create a closure that we don't actually need